src/main/frontend/app/App.js

```diff
@@ -0,0 +1,63 @@
+import React, {Component} from 'react';
+import ReviewFormContainer from './containers/ReviewFormContainer'
+
```

Comment:
should we give this high level component a better name than `App`? Let's make it more descriptive